po/README_TRANSLATORS.md

```diff
@@ -0,0 +1,148 @@
+# Localizing Ghostty: The Translators' Guide
+
+First of all, thanks for helping us localize Ghostty!
+
+To begin contributing, please make sure that you have installed the `gettext`
+package on your system, which should be available under the name `gettext`
+for most Linux and macOS package managers.
```

Comment:
Subjective, but calling `gettext` a package didn't feel right to me:
```suggestion
To begin contributing, please make sure that you have `gettext` installed on
your system, which should be available under the name `gettext` for most Linux
and macOS package managers.
```
Feel free to resolve this comment if you disagree.

Reply:
It's to differentiate gettext the tool and gettext the thing you can install. Maybe could be rephrased better